docs/pages/module.js

```diff
@@ -582,3 +523,19 @@ function ModuleExample5() {
     </Page>
   );
 }
+
+export async function getStaticProps(): Promise<{| props: {| generatedDocGen: DocGen |} |}> {
+  const generatedDocGen = await docgen('Module');
+
+  generatedDocGen.props.icon = {
+    ...generatedDocGen.props.icon,
+    flowType: {
+      name: 'string',
+      raw: 'Icon[icon]',
```

Comment:
<img width="884" alt="Screen Shot 2021-12-13 at 11 14 13 AM" src="https://user-images.githubusercontent.com/12059539/145873847-6718fa21-8aae-4f3f-9b8a-a370cd418e8a.png">

The current `string` type for this prop isn't helpful, but I'm not sure that this is either. Perhaps the description should include a link to `/icon` to guide the user to valid icon names?

Reply:
@dangerismycat This uses Flow Indexed Access types: https://flow.org/en/docs/types/indexed-access/. I also added a link to https://gestalt.netlify.app/iconography_and_svgs

Reply:
TIL!